Sources/Player/OfflineEngine/Internal/Storage/GRDBStorage/OfflineEntryGRDBEntity.swift

```diff
@@ -109,4 +106,22 @@ struct OfflineEntryGRDBEntity: Codable, FetchableRecord, PersistableRecord {
 		mediaBookmark = try? entry.mediaURL?.bookmarkData()
 		licenseBookmark = try? entry.licenseURL?.bookmarkData()
 	}
+
+	func bookmarkDataNeedsUpdating() -> Bool {
+		do {
+			var mediaBookmarkIsStale = false
+			var licenseBookmarkIsStale = false
+
+			if let mediaBookmark {
+				_ = try URL(resolvingBookmarkData: mediaBookmark, bookmarkDataIsStale: &mediaBookmarkIsStale)
+			}
+			if let licenseBookmark {
+				_ = try URL(resolvingBookmarkData: licenseBookmark, bookmarkDataIsStale: &licenseBookmarkIsStale)
+			}
+
+			return mediaBookmarkIsStale || licenseBookmarkIsStale
+		} catch {
+			return false
```

Comment:
[question] just checking in case it would be relevant to have logging here. Since we would get false in such case, and the continue flow would not succeed I suppose.